rest/main/main.go

```diff
@@ -105,8 +105,6 @@ func main() {
 		Certificates:             prepareServerCertificate(),
 		ClientCAs:                prepareCACertificates(),
 		MinVersion:               tls.VersionTLS12,
-		PreferServerCipherSuites: true,
```

Comment:
Are you planning to upgrade golang version too? If not it is better to retain the **PreferServerCipherSuites** setting. Without this, the golang 1.15 tls library selects client advertised cipher suite. Hence there is a chance of using an unsafe cipher if the clients are not configured properly.

Reply:
I will upgrade golang 1.15 and keep disable PreferServerCipherSuites settings

Reply:
After discussion, keep Go 1.13 and retain PreferServerCipherSuites. only remove CipherSuites.